### PR TITLE
EN feature SAP logs on OVHcloud Logs Data Platform

### PR DESCRIPTION
**Pull request by antoinemarneur**
Hi team,

We have created new guides to help customers configure their SAP systems to forward SAP logs to the OVHcloud Logs Data Platform.
These guides are composed of two main themes: Configure and Analyze, with an additional page displaying the different tags available for each log.
The guide related to the system configuration will also be displayed on the OVHcloud Logs Data Platform guide homepage.

Antoine,

### CODE REVIEW

**Comment by OVH-Team-Guides** [comment]
NOT TO MERGE before formal validation from @antoinemarneur / @mlt-ovh / @remrozk

File: pages/hosted_private_cloud/sap_on_ovhcloud/cookbook_sap_logs_on_ovhcloud_logs_data_platform_analyze_and_work_with_your_logs/guide.de-de.md [hunk 1]
```diff
@@ -1,13 +1,128 @@
...
+| saplog      | Defines the category of your log.<br>Examples: syslog, security_audit, dev_w, nameserver, etc. |
+| source      | Hostname of the server which forwarded the log. |
+| timestamp   | Timestamp (UTC). |
+
+Examples of filter usage:
+
+<details>
+<summary>Logs of the work process trace files (dev_w*)</summary>
+
+![filter_dev_w1](images/filter_dev_w1.png){.thumbnail}
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/sap_on_ovhcloud/cookbook_sap_logs_on_ovhcloud_logs_data_platform_analyze_and_work_with_your_logs/guide.de-de.md]: These images won't show. The `<details>`is HTML so it implies everything in it is in HTML too.
If you want to keep this collapsible display, you need to use HTML such as `alt-text`

**Comment by remrozk** [comment]
Hi @OVH-Team-Guides 

You can merge this PR 

Thank you!
-- Rémi

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @antoinemarneur & @remrozk

**Pull request merged**